### ISSUE TITLE
[cloud-tensorflow-mnist]:Two issues in codelab

### ISSUE THREAD

**Issue by dansitu** [googlecodelabs/feedback#337]
On step 7, it is not possible to beat 97% accuracy without increasing training iterations beyond 2000 on line 111, but the text doesn't mention anything about iterations yet.

On step 8, the softmax_cross_entropy_with_logits function requires named arguments (labels= and logits=) but the example given in the codelab doesn't show this.
